src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -1591,7 +1591,7 @@ private static SocketError SendFileAsync(SafeCloseSocket handle, FileStream file
             Socket socket, TransmitFileOptions options, SendPacketsElement[] elements, FileStream[] files, Action<long, SocketError> callback)
         {
             SocketError error = SocketError.Success;
-            long bytesTransferred = 0;
+            long[] bytesTransferred = {0};
```

Comment:
Why is this an array?

Reply:
Because it changed inside closure. There was warning about it and suggestion to convert to array.

Reply:
It shouldn't need to be an array.  What warning are you seeing, exactly?

Reply:
https://www.jetbrains.com/help/resharper/2017.3/AccessToModifiedClosure.html

Reply:
Converting it to an array won't address what it's flagging.  And what it's flagging isn't actually a problem in this code.  This shouldn't be an array.